apps/publisher/modules/page-decorators.js

```diff
@@ -205,13 +205,13 @@ var pageDecorators = {};
             sortObj.active = false;
             sortObj.sortNext = "+";
             sortObj.sortIcon = "sorting_asc";
-            if(sortable[i].field == sortBy){
-                if(sort == "+"){
-                    sortingListSelected.helpIcon = "sorting_asc";
+            if(sortable[i].field == sortBy) {
+                if(sort == "+") {
+                    sortingListSelected.helpIcon = "fw-sort-up margin-bottom-align";
                     sortObj.sortNext = "-";
                     sortObj.sortIcon = "sorting_asc";
-                }else if(sort == "-"){
-                    sortingListSelected.helpIcon = "sorting_desc";
+                } else if(sort == "-") {
```

Comment:
Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/94c71e676e222f35e133ecddcb38d0c234247b62)